src/main/java/io/cryostat/net/reports/ActiveRecordingReportCache.java

```diff
@@ -67,29 +67,44 @@
 class ActiveRecordingReportCache implements NotificationListener<Map<String, Object>> {
     protected final Provider<ReportGeneratorService> reportGeneratorServiceProvider;
     protected final FileSystem fs;
-    protected final LoadingCache<RecordingDescriptor, String> cache;
+    protected final LoadingCache<RecordingDescriptor, String> htmlCache;
+    protected final LoadingCache<RecordingDescriptor, String> jsonCache;
     protected final TargetConnectionManager targetConnectionManager;
     protected final long generationTimeoutSeconds;
+    protected final long cacheExpirySeconds;
+    protected final long cacheRefreshSeconds;
+
     protected final Logger logger;
 
     ActiveRecordingReportCache(
             Provider<ReportGeneratorService> reportGeneratorServiceProvider,
             FileSystem fs,
             TargetConnectionManager targetConnectionManager,
             @Named(ReportsModule.REPORT_GENERATION_TIMEOUT_SECONDS) long generationTimeoutSeconds,
+            @Named(ReportsModule.ACTIVE_REPORT_CACHE_EXPIRY_SECONDS) long cacheExpirySeconds,
+            @Named(ReportsModule.ACTIVE_REPORT_CACHE_REFRESH_SECONDS) long cacheRefreshSeconds,
             Logger logger) {
         this.reportGeneratorServiceProvider = reportGeneratorServiceProvider;
         this.fs = fs;
         this.targetConnectionManager = targetConnectionManager;
         this.generationTimeoutSeconds = generationTimeoutSeconds;
+        this.cacheExpirySeconds = cacheExpirySeconds;
+        this.cacheRefreshSeconds = cacheRefreshSeconds;
         this.logger = logger;
-        this.cache =
+        this.htmlCache =
                 Caffeine.newBuilder()
                         .scheduler(Scheduler.systemScheduler())
                         .expireAfterWrite(30, TimeUnit.MINUTES)
                         .refreshAfterWrite(5, TimeUnit.MINUTES)
                         .softValues()
-                        .build((k) -> getReport(k));
+                        .build((k) -> getReport(k, "", true));
+        this.jsonCache =
+                Caffeine.newBuilder()
+                        .scheduler(Scheduler.systemScheduler())
+                        .expireAfterWrite(cacheExpirySeconds, TimeUnit.SECONDS)
+                        .refreshAfterWrite(cacheRefreshSeconds, TimeUnit.SECONDS)
+                        .softValues()
+                        .build((k) -> getReport(k, "", false));
```

Comment:
Maybe worth adding an overload for `getReport(k, false)` that passes this empty filter implicitly. And perhaps a named constant `NO_FILTERS = ""` would be useful for readability too?